packages/nx/src/command-line/watch/watch.ts

```diff
@@ -48,60 +49,77 @@ class BatchCommandRunner {
     return this.process();
   }
 
-  async process() {
+  private async process() {
```

Comment:
At this point `enqueue` is just `process`, can we move the body into `enqueue` and remove `process`?

Reply:
Github hid it but `enqueue` also adds projects and files into pending